python/lsst/afw/image/apCorrMap.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import absolute_import, division, print_function
+
+from ._apCorrMap import ApCorrMap
+
+ApCorrMap.values = lambda self: [self[name] for name in self.keys()]
+ApCorrMap.items = lambda self: [(name, self[name]) for name in self.keys()]
+ApCorrMap.__iter__ = lambda self: iter(self.keys())
```

Comment:
I agree that this is the best way to do this, but I think the style guide generally forbids using lambdas. Although one could argue that in this case the functions are very clear and it is a much simpler implementation.